### PR TITLE
Feature/idenfy

### PR DESCRIPTION
**Pull request by SarunasAzna**

### CODE REVIEW

File: src/components/Order/OrderMain/OrderState/OrderFiatModals/KYCModalTier0/KYCModalTier0.jsx [hunk 1]
```diff
@@ -168,24 +168,33 @@ class KYCModal extends Component {
...
               {this.props.kyc.residence_document_status !== 'APPROVED' && (
                 <div>
                   <h2>{t('order.fiat.kyc.2')}</h2>
                   <small>
-                    {t('order.fiat.kyc.21')}
+                      <div dangerouslySetInnerHTML={{__html: t('order.fiat.kyc.21')}} />
                   </small>
                   <small>
-                    {t('order.fiat.kyc.22')}
+                      <div dangerouslySetInnerHTML={{__html: t('order.fiat.kyc.22')}} />
```
Comment (SarunasAzna) [inline src/components/Order/OrderMain/OrderState/OrderFiatModals/KYCModalTier0/KYCModalTier0.jsx]: this was a bug (html from translations was not working)

File: src/components/Order/OrderMain/OrderState/OrderPaymentFiat/OrderPaymentTemplateFiat/DesktopNotifications/DesktopNotifications.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ class DesktopNotifications extends Component {
       <div className="row">
         <div className="col-xs-12 text-center">
           <a href="javascript:void(0)" className={styles.title} onClick={this.onClick}>
-            <h4>{t('order.kycstatuscheck')}</h4>
+            <h4>{t('order.fiat.kyc.statuscheck')}</h4>
```
Comment (SarunasAzna) [inline src/components/Order/OrderMain/OrderState/OrderPaymentFiat/OrderPaymentTemplateFiat/DesktopNotifications/DesktopNotifications.js]: this was a bug too (no such term order.kycstatuscheck)

File: src/components/Order/OrderMain/OrderState/OrderFiatModals/KYCModalTier0/KYCModalTier0.jsx (hunk 1 above)
Comment (SarunasAzna) [reply src/components/Order/OrderMain/OrderState/OrderFiatModals/KYCModalTier0/KYCModalTier0.jsx]: note: IMHO call terms 21 and 22 is a bad practice

Comment (BeOleg) [reply src/components/Order/OrderMain/OrderState/OrderFiatModals/KYCModalTier0/KYCModalTier0.jsx]: Yeah, agreed

**Pull request merged**